### ISSUE TITLE
Getting an error  on "from langchain.chains import RetrievalQA"; error message is cannot import name 'NeptuneRdfGraph' from 'langchain_community.graphs' .  using langcain version 0.1.7

### ISSUE THREAD

**Issue by NarayananParthasarathy** [langchain-ai/langchain#17680]
Getting an error  on "from langchain.chains import RetrievalQA"; error message is cannot import name 'NeptuneRdfGraph' from 'langchain_community.graphs' .  using langcain version 0.1.7

_Originally posted by @NarayananParthasarathy in https://github.com/langchain-ai/langchain/issues/2725#issuecomment-1950262407_

**Comment by NarayananParthasarathy** [comment]
Hi ,
Thanks for the reply. However, I'm NOT using the NeptuneGraph at all. I just need to import RetrievalQA from langchain.chains & that is not working because of this graph issues. How so I resolve the langchain.chains import RetrievalQA ?

**Comment by NarayananParthasarathy** [comment]
@dosu-bot :
After updating my langchain version to 0.1.0, the issue is resoled.  I was having langchain-community to 0.0.19 which could have caused a conflict with langchain 0.1.7.

**Comment by diegoquirosm** [comment]
I was having the same issue. Following dependencies solve it. langchain==0.1.8, langchain-community==0.0.19

**Comment by romilan24** [comment]
TLDR, try: pip3 uninstall langchain, pip3 install langchain . . and then as @dosubot suggested: pip install -U langchain-community

This combination worked for me ty.

**Comment by prathigajd** [comment]
I'm facing this Issue. Even though when am using langfuse==v1.8.0, langchain==0.1.0, langchain-community==0.0.19. 

It would be great if anyone finds a solution


───────────────── Traceback (most recent call last) ──────────────────╮
                             │ C:\Users\SIT-100033-US\Desktop\python_langchain\langflow\lib\site-pac │
                             │ kages\langflow\services\chat\service.py:200 in handle_websocket       │
                             │                                                                       │
                             │   197 │   │   │   await websocket.send_json(chat_history)             │
                             │   198 │   │   │                                                       │
                             │   199 │   │   │   while True:                                         │
                             │ ❱ 200 │   │   │     json_payload = await websocket.receive_json()     │
                             │   201 │   │   │     if isinstance(json_payload, str):                 │
                             │   202 │   │   │   │   │   payload = orjson.loads(json_payload)        │
                             │   203 │   │   │     elif isinstance(json_payload, dict):              │
                             │                                                                       │
                             │ C:\Users\SIT-100033-US\Desktop\python_langchain\langflow\lib\site-pac │
                             │ kages\starlette\websockets.py:139 in receive_json                     │
                             │                                                                       │
                             │   136 │   │   │   │   'WebSocket is not connected. Need to call "acce │
                             │   137 │   │   │   )                                                   │
                             │   138 │   │   message = await self.receive()                          │
                             │ ❱ 139 │   │   self._raise_on_disconnect(message)                      │
                             │   140 │   │                                                           │
                             │   141 │   │   if mode == "text":                                      │
                             │   142 │   │   │   text = message["text"]                              │
                             │                                                                       │
                             │ C:\Users\SIT-100033-US\Desktop\python_langchain\langflow\lib\site-pac │
                             │ kages\starlette\websockets.py:111 in _raise_on_disconnect             │
                             │                                                                       │
                             │   108 │                                                               │
                             │   109 │   def _raise_on_disconnect(self, message: Message) -> None:   │
                             │   110 │   │   if message["type"] == "websocket.disconnect":           │
                             │ ❱ 111 │   │   │   raise WebSocketDisconnect(message["code"], message. │
                             │   112 │                                                               │
                             │   113 │   async def receive_text(self) -> str:                        │
                             │   114 │   │   if self.application_state != WebSocketState.CONNECTED:  │
                             ╰───────────────────────────────────────────────────────────────────────╯
                             WebSocketDisconnect: (1012, None)

**Comment by smv9495** [comment]
Try using:
```
from langchain.chains.retrieval_qa.base import RetrievalQA
```
